CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ Main (unreleased)
 - Grafana Agent Operator: add support for integrations through an `Integration`
   CRD which is discovered by `GrafanaAgent`. (@rfratto)
 
+- (beta) Add app agent receiver integration. This depends on integrations-next being enabled
```

Comment:
Thoughts on labeling experimental? The meaning behind beta is a bit broad but in general, BETA indicates we plan to move to production where experimental is much more fluid. This feels experimental. @rfratto thoughts?

Reply:
I think that's going to be up to @domasx2. It sounds like @domasx2 and team are confident this is going to be something that gets delivered as stable, so calling it beta makes sense IMO.

Reply:
hm, I think let's make it experimental at least while we dogfood and build confidence
- [ ] experimental